code/game/objects/items/RCD.dm

```diff
@@ -643,8 +637,6 @@ GLOBAL_VAR_INIT(icon_holographic_window, init_holographic_window())
 	..()
```

Comment:
Oh!

It's because this is `..()` and not `. = ..()`

Meaning you never actually get the fail value

(You might not even need ui_host() at all, give that a try)

Reply:
Make sure RPDs do this too

Reply:
oh my god. give me a sec

Reply:
This would be awesome to lint for but SpacemanDMM doesn't give us that capability yet, wonder what else doesn't have this

Reply:
Nope it didnt work :(. You definitly caught a bug thanks for that but ghosts can still have their fun with the RCD and the console still needs `ui_host()` to open

It also makes sense why it does not work because if you look inside RPD` ui_act()`
```
. = ..()
	if(.)
		return
	if(!usr.canUseTopic(src, be_close = TRUE))
		return
```
You will they didnt make the same mistake as me and still used `canUseTopic()`

Im going no where fast